frontend/src/components/navigationWhite/navigationWhite.js

```diff
@@ -1,41 +1,36 @@
-import React, { useEffect, useRef } from "react";
+import React, { useRef, useState } from "react";
 import "./navigationWhite.css";
 import logo from "../../images/auction-app-logo 1.png";
 import { useNavigate, useLocation } from "react-router-dom";
 
 function NavigationWhite() {
   const user = JSON.parse(localStorage.getItem("user"));
-  let searchField = useRef();
   let navigate = useNavigate();
   const location = useLocation();
   const showExitButton = useRef(false);
+  const [searchField, setSearchField] = useState("");
   const showSearchAndMenu = useRef(
-    location.pathname.includes("/login") ||
-      location.pathname.includes("/register")
-      ? false
-      : true
+    !location.pathname.includes("/login") &&
```

Comment:
Should this be without negation or not?

Reply:
Both should be with negative because if pathname is not login and it is not register, then showSearchAndMenu should be true, but in those two cases it will be true and it will show only logo